xml/bp_troubleshooting_cephadm.xml

```diff
@@ -60,6 +60,10 @@
  </sect1>
  <sect1 xml:id="bp-troubleshooting-cephadm-logs">
    <title>Accessing &ceph; Daemon Logs</title>
+   <para>
+     &productname; &productnumber; supports &ceph; logging via systemd-journald. To access
+     the logs of &ceph; daemons in  &productname; &productnumber;, follow the next instructions.
```

Comment:
Follow the proceeding instructions? Hm, @lproven what do you think?

Reply:
"To learn how to access the logs of Ceph daemons, read the following text."

"The procedure for accessing the logs of Ceph daemons is described below."

(could probably think of more?)

Reply:
Described below follows more along with what we'd normally say :thinking:  my brain keeps going "follow the following instructions" like it's on repeat

Reply:
"follow the instructions, below" is another option

Reply:
or "use the following procedure"